improver_tests/acceptance/SHA256SUMS

```diff
@@ -212,9 +212,11 @@ ea67ae7a7f5363ae3692b29c93fb9989449d96e49dc613a1297d98ddb12c578a  ./categorical-
 8c27cd09164c1fc18d6e7474746867f7a4680a1388b7057437f995663c4de26d  ./categorical-modes/spot_ties/kgo.nc
 2ec32b8654824a633fcc9d7405d4d3a8b487514bddb9e8ec6aae66c9a96c2565  ./categorical-modes/wx_decision_tree.json
 d8c2fadd946060703615db0e6cd1c7a0a0186c2d19c685b77a0f6b043d5b5e8a  ./categorical/bad_wx_decision_tree.json
+98a3556ebb5e2c041b9bd6e222818bba52b2e4f216a902fe26f69207699f34da  ./categorical/basic/cloud_cubelist.nc
 c2fdb1144abdf6188bb2d9de0104c6dfbcb3a8ba07cbe4e7ac2b9c3782f55484  ./categorical/basic/kgo.nc
 9c543a6e2068ad51c6664903bf06b5ecbf018ad914f972762ef0577469fec1b3  ./categorical/basic/kgo_no_lightning.nc
 fabc3622379b8d2b1b1595b6e6be658dbfb30214ca9516e61272973b211ca75f  ./categorical/basic/kgo_titled.nc
+951a8e75e3ad944fa654117818c4dbe6778e63340e8c007dd8ed2113fd1b4090  ./categorical/basic/param_cubelist.nc
```

Comment:
Why do these two fields show up as changes? They weren't added in the acceptance data PR.

Reply:
These were old cubes I was trying to make work. I had deleted them from the acceptance test data branch but must have done that after creating the checksums. I've re-created the checksums and these files are gone now. The tests all still pass.